Sources/Sentry/Public/SentryTransactionContext.h

```diff
@@ -56,6 +61,17 @@ SENTRY_NO_INIT
                    operation:(NSString *)operation
                      sampled:(SentrySampleDecision)sampled;
 
+/**
```

Comment:
Same as in https://github.com/getsentry/sentry-cocoa/pull/4751#discussion_r1930846919